specification/consumption/resource-manager/readme.md

```diff
@@ -78,3 +87,10 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 ``` yaml $(tag) == 'package-2017-04-preview' && $(go)
 output-folder: $(go-sdk-folder)/services/consumption/mgmt/2017-04-24-preview/consumption
 ```
+
+### Tag: package-2017-11-30 and go

```

Comment:
@marstr Please confirm this new tag is ok

Reply:
Confirmed! Thanks for adding this, @sandeepnl. Thanks for looping me in, @lmazuel!